.evergreen/download-mongodb.sh

```diff
@@ -58,6 +58,8 @@ get_mongodb_download_url_for ()
    VERSION_70="7.0.0-rc0"
    VERSION_60_LATEST="v6.0-latest"
    VERSION_60="6.0.6"
+   # The perf version must always remain pinned to the same patch
+   VERSION_60_PERF="6.3.1"
```

Comment:
Should this be 6.0.6 since it's VERSION_60_PERF?

Reply:
Yeah that's a good point, I was autopiloting to match what node is currently testing against, which is rapid, but I think it makes the most sense to use the "official" current version instead, and that's what we'll likely be adding for 7.0 testing going forward, too.